tf-modules/cumulus-test-cleanup/README.md

```diff
@@ -0,0 +1,20 @@
+# Cumulus Test Cleanup
+
+Module to run nightly test cleanup of resources left over from Cumulus integration test runs. This deploys a test cleanup lambda that is configured to run nightly.
+
+This is meant to be deployed once for each AWS account.
+
+Currently cleans up:
+- Kinesis streams generated by tests that are older than one day
+
+## Build
+
+Run `npm install` to install packages then `npm run prepare` to prepare lambda zip file for deployment.
+
+## Deployment
+
+Copy the `terraform.tfvars.sample` file to `terraform.tfvars`. Populate the sample file with values that apply to your AWS environment.
```

Comment:
If we're using this as a team, do we want to have a state configuration/ store state remotely so others could create/tear down using terraform so we're not leaving policies/etc hanging?

Reply:
right right. Ok so I do have it in remote state AND in such a way that it would work nicely with my deployment report. Should I commit the `terraform.tf`? Or put it in the README? The other thing I am thinking is I should deploy it to SIT so maybe not commit the `terraform.tf`.

Reply:
Hmm, not sure on that point.   We deliberately don't have a terraform.tf in the cumulus-tf module, right?       Probably README instructions would suffice in either case.

Reply:
I did update the README. Let me know if you think that is enough.